cmd/buildah/config.go

```diff
@@ -135,13 +135,15 @@ func updateConfig(builder *buildah.Builder, c *cli.Context) {
 		}
 	}
 	if c.IsSet("entrypoint") {
-		entrypointSpec, err := shellwords.Parse(c.String("entrypoint"))
-		if err != nil {
-			logrus.Errorf("error parsing --entrypoint %q: %v", c.String("entrypoint"), err)
-		} else {
-			builder.SetEntrypoint(entrypointSpec)
-			builder.SetCmd(nil)
+		entrypointSpec := make([]string, 3)
+		entrypointSpec[0] = "/bin/sh"
+		entrypointSpec[1] = "-c"
+		entrypointSpec[2] = c.String("entrypoint")
+		builder.SetEntrypoint(entrypointSpec)
```

Comment:
Nit:  Should this be the else clause of the if statement below.  Saves a command.